scripts/spack/packages/axom/package.py

```diff
@@ -177,12 +177,14 @@ class Axom(CachedCMakePackage, CudaPackage, ROCmPackage):
         depends_on(f"raja {ext_cuda_dep}", when=f"+raja {ext_cuda_dep}")
         depends_on(f"umpire {ext_cuda_dep}", when=f"+umpire {ext_cuda_dep}")
         depends_on(f"caliper {ext_cuda_dep}", when=f"+profiling {ext_cuda_dep}")
+        depends_on(f"mfem {ext_cuda_dep}", when=f"+mfem {ext_cuda_dep}")
```

Comment:
👍

Reply:
Does this resolve https://github.com/LLNL/axom/issues/1514 ?

Reply:
We will soon -- I'll be integrating the new GPU InverseNewton solve into our point in cell query (https://github.com/mfem/mfem/pull/4701)